views/sidebars/carousel.handlebars

```diff
@@ -1,19 +1,52 @@
 <a href="#" class="backButton button btn-secondary"><i class="fa fa-back"></i> Back</a>
 
-<h5>Image 1</h5>
-
-<form method="PUT" action="/api/customization/carousel" class="customizationForm">
+<form  method="PUT" action="/api/customization/carousel/{{index}}" class="customizationForm {{data.option}}">
 
-<input name="image-one" class="form-control mb-3"  value="" />
```

Comment:
We need to keep this.  It should look like this.
```<input name="image-one" class="form-control mb-3"  value="{{#if data.customization.image-one}}{{ data.customization.image-one}}{/if}" />```

Reply:
Ok Updating now Thanks!